.changelogs/fix_lifterlms#2174.yml

```
@@ -0,0 +1,9 @@
+significance: patch
+type: fixed
+comment: Added condition for checking if the post is an auto-draft and the
+  Classic Editor is not active when postmeta is not generated for hiding two
+  options for gutenberg editor.
+links:
+  - "lifterlms#2174"
```

```suggestion
links:
  - "lifterlms#2174"
```
plain link here, cross-repository linking doesn't work :D

---

I think the plain link also doesn't work. The Changelog validator rejects the plain link and the cross-links. Just the issue numbers.